mitodl/bootcamp-ecommerce
# Add uWSGI settings

#### What are the relevant tickets?

https://trello.com/c/upfutK0A/44-update-all-uwsgi-configs-with-recommendations-from-bloomberg-article

#### What's this PR do?

It will add uWSGI settings to override the software's defaults.

See https://www.techatbloomberg.com/blog/configuring-uwsgi-production-deployment/
See https://github.com/mitodl/cookiecutter-djangoapp/pull/163


#### How should this be manually tested?

No manual testing. If uWSGI runs in CI or RC it should validate the configuration.

┆Attachments: <a href="https://github.com/mitodl/bootcamp-ecommerce/pull/358">bootcamp-ecommerce #358 Add uWSGI settings</a>
